apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -32,10 +81,123 @@ for (const [i, transaction] of transactions.entries()) {
+      testName: `sendToken_${transaction.accountToDebit.currency.uiName}`,
+      speculosCurrency: specs[transaction.accountToDebit.currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: i,
+    });
+
+    test(`[${transaction.accountToCredit1.accountName}] send token`, async ({ page }) => {
+      addTmsLink(["B2CQA-479"]);
+
+      const app = new Application(page);
+
+      await app.layout.goToAccounts();
+      await app.accounts.navigateToAccountByName(transaction.accountToDebit.accountName);
+      await app.account.navigateToTokenInAccount(transaction.token);
+      await app.account.clickSend();
+      await app.send.fillRecipient(transaction.accountToCredit1.address);
+      await app.send.checkASAError();
+      await app.send.fillRecipient(transaction.accountToCredit2.address);
+      await app.send.checkContinueButtonEnable();
+    });
+  });
+}
+
+for (const [i, transaction] of transactionETH_BTC.entries()) {
+  test.describe.parallel("ETH input error", () => {
+    test.use({
+      userdata: "speculos-tests-app",
+      testName: `ETH_input_error_${transaction.accountToDebit.currency.uiName}`,
+      speculosCurrency: specs[transaction.accountToDebit.currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: i,
+    });
+
+    test(`[${transaction.accountToDebit.accountName}] ETH input error`, async ({ page }) => {
+      addTmsLink(["B2CQA-1904"]);
+
+      const app = new Application(page);
+
+      await app.layout.goToAccounts();
+      await app.accounts.navigateToAccountByName(transaction.accountToDebit.accountName);
+
+      await app.account.clickSend();
+      await app.send.fillRecipient(transaction.accountToCredit.address);
+      await app.send.checkInvalidAddressError(transaction);
+    });
+  });
+}
+
+for (const [i, transaction] of transactionsInputsInvalid.entries()) {
+  test.describe.parallel("send test - invalid input", () => {
+    test.use({
+      userdata: "speculos-tests-app",
+      testName: `Send_test_invalid_input_${transaction.transaction.accountToDebit.currency.uiName}`,
+      speculosCurrency:
+        specs[transaction.transaction.accountToDebit.currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: i,
+    });
+
+    test(`[${transaction.expectedErrorMessage}] Send test - Invalid input (${transaction.transaction.accountToDebit.currency.uiName}) - ${i}`, async ({
+      page,
+    }) => {
+      addTmsLink(["B2CQA-473"]);
+
+      const app = new Application(page);
+
+      await app.layout.goToAccounts();
+      await app.accounts.navigateToAccountByName(
+        transaction.transaction.accountToDebit.accountName,
+      );
+
+      await app.account.clickSend();
+      await app.send.fillRecipient(transaction.transaction.accountToCredit.address);
+      await app.send.clickContinue();
+      await app.send.fillAmount(transaction.transaction);
+      await app.send.checkErrorMessage(transaction.expectedErrorMessage);
+      await app.send.checkContinueButtonDisabled();
```

Comment:
you check that continue button is disabled 2 times: in this method and in the previous one as well